code/gamesnd/gamesnd.cpp

```diff
@@ -860,9 +861,11 @@ void parse_gamesnd_new(game_snd* gs, bool no_create)
 	// Default pitch is 1.0. This is set here in case we don't have a valid file name
 	gs->pitch_range = util::UniformFloatRange(1.0f);
 
-	if (!stricmp(name, NOX("empty")))
+	if (!stricmp(name, NOX("empty")) || !stricmp(name, NOX("none")))
 	{
 		entry->filename[0] = 0;
+		advance_to_eoln(nullptr);
```

Comment:
Hmm, I wonder why this line wasn't there before. Was it just missing?

Reply:
Hmm, good question.  I don't think it was missing because new gamesnds are parsed differently.  I'll take it out.